tools/ocean_data_view_manager/tool_odv.xml

```diff
@@ -0,0 +1,182 @@
+            <option value="R_APP_QV">QV that creates a odv collection with raw and ref files to qualify</option>
+            <option value="DEMO">Valide with ODV demo that performes the demo</option>
+        </param>
+        <conditional name="config">
+            <param name="type" type="select" label="Select if you want to write your own configuration file or not.">   
+                <option value="no">No, I don't want to, I'll use the default one</option>
+                <option value="yes">Yes, I to write my own configuration file</option>
+            </param>
+            <when value="yes">
+                <param name="operator" type="text" value="Anonymous" label="Enter operator name"/>
+                <param name="qc_convention" type="select" label="Enter QC convention for the ODV collection output (default: ARGO)">
+                    <option value="ARGO">ARGO</option>
+                    <option value="ODV">ODV</option>
+                    <option value="GTSPP">GTSPP</option>
+                    <option value="SEADATANET">SEADATANET</option>
+                    <option value="ESEAS">ESEAS</option>
+                    <option value="WOD">WOD</option>
+                    <option value="WODSTATION">WODSTATION</option>
+                    <option value="WOCEBOTTLE">WOCEBOTTLE</option>
+                    <option value="WOCECTD">WOCECTD</option>
+                    <option value="QARTOD">QARTOD</option>
+                    <option value="BODC">BODC</option>
+                    <option value="PANGAEA">PANGAEA</option>
+                    <option value="SMHI">SMHI</option>
+                    <option value="OceanSITES">OceanSITES</option>
+                    <option value="IODE">IODE</option>
+                </param>
+                <param name="subsetting" type="float" min="-1" max="1" value="1" label="Enter subsetting (default: 1)"/>
+                <param name="plt" type="float" min="0" max="1" value="0" label="Enter plt (default: 0 for QV, 1 for demo)"/>
+            </when>
+            <when value="no">
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="output" format="txt" label="ODV tool output"/>
+        <collection type="list" name="files" label="ODV tool collection">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)" directory="./" recurse="true" format="txt"/>
+        </collection>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="2">
+            <param name="input_raw" location="https://gitlab.com/pokapok-projects/easy-qcv-bgc/qcv-odv-tool/-/raw/7b3557c89438b7347c601d6959af354103b167a1/galaxy_tool/test-data/data_raw.nc"/>
+            <param name="input_ref" location="https://gitlab.com/pokapok-projects/easy-qcv-bgc/qcv-odv-tool/-/raw/7b3557c89438b7347c601d6959af354103b167a1/galaxy_tool/test-data/data_ref.nc"/>
+            <param name="mode" value="R_APP_QV"/>
+            <param name="type" value="no"/>
```

Comment:
```suggestion
            <conditional name="">
                <param name="mode" value="R_APP_QV"/>
                <param name="type" value="no"/>
            </conditional>
```

something like that is now needed ... please note that the galaxy-language-server is generating you test-templates if you like :)

Reply:
On it ! will improve that tomorrow ! Dankeschön